src/status_im/utils/keychain.cljs

```diff
@@ -0,0 +1,55 @@
+(ns status-im.utils.keychain
+  (:require [cljs.core.async :as async]
```

Comment:
Please don't use core.async, if you need asynchronous behaviours use re-frame dispatches instead

Reply:
Any reason? It's used in several other places in the project